docs/pages/design/alerts.mdx

```diff
@@ -57,7 +57,7 @@ Links within alerts are set in 18px HK Grotesk SemiBold text (check [typography]
 
 <CodeExampleBox p="md" pb="xxs">
 <Alert variant="success" mb="md">
-    This is a success alert! This is an accompanying  <Link href="#">link</Link>.
+    This is a success alert! This is an accompanying  <Link href="#">{" link"}</Link>.
```

Comment:
to me it looks like this still didn't fix the issue - but i see you've put one in #83

Reply:
yeah im not sure how to fix it